ddl/ddl.go

```diff
@@ -60,7 +60,7 @@ var (
 	// EnableSplitTableRegion is a flag to decide whether to split a new region for
 	// a newly created table. It takes effect only if the Storage supports split
 	// region.
-	EnableSplitTableRegion = true
+	EnableSplitTableRegion = false
```

Comment:
Why disable split?

Reply:
This value only affects unit tests, for unit test there is no need to split.
If run as a server, the config is used instead which is enabled by default.

If region is split, we will backoff 100ms.